### DISCUSSION TITLE
Extra data to Middleware/Remove a Middleware

### DISCUSSION THREAD

**Discussion by paulosalvatore** [prisma/prisma#6101] [Q&A]
## Question

I don't think that is possible yet, but how can I remove a `Middleware` that was added by `prisma.$use`?

I know I can set the function just to return `next(params)` without any side-effect, but this doesn't seem right because the function would still be called.

## My use case

I'm trying to get specific data that is related to some `prisma` function that was called, but I don't want to send it to database, just want to check a few things inside a middleware.

**Comment by ryands17** [comment]
@paulosalvatore 👋 

I would suggest creating a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) with your use case so that we can look into this. The workaround that you have specified is the only way to do this for now.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
